src/sage/rings/algebraic_closure_finite_field.py

```diff
@@ -833,7 +833,7 @@ def gen(self, n):
         F = self._subfield(n)
         return self(F.gen())
 
-    def gens(self):
+    def gens(self):  # -> Family
```

Comment:
Why is this commented-out? Looks correct to me. 

If it's because you don't want to add a global import, you can add the import with an additional check for `TYPE_CHECKING`: https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
because `Family` is not a class

Reply:
Right, so it should be `AbstractFamily`?

Reply:
right, done